sdk.sln

```diff
@@ -63,10 +63,18 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.NET.Build.Extensi
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.NET.Build.Extensions.Tasks.UnitTests", "src\Tasks\Microsoft.NET.Build.Extensions.Tasks.UnitTests\Microsoft.NET.Build.Extensions.Tasks.UnitTests.csproj", "{0E11CA1E-D64F-48E9-BE86-B040D9E8EBF8}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "test", "{4A305861-DAB7-455A-8A0F-27F82D79D971}"
+EndProject
```

Comment:
Oh I see, it's a sln folder. Open things up in VS and see how it looks. I think you'll end up with this test project in a folder, but not the others. If you want them all to be in a folder, that's OK, but let's be consistent